When i upgrade angular version 1.5.0-rc.2  then it is not working

angular.js:13307 Error: [ui.select:transcluded] Expected 1 .ui-select-match but got '0'.
    at http://test.com:9012/library/ui-select/dist/select.js:125:12
    at http://test.com:9012/library/ui-select/dist/select.js:1035:19
    at publicLinkFn (http://test.com:9012/bower_components/angular/angular.js:8193:29)
    at http://test.com:9012/bower_components/angular/angular.js:8535:29
    at boundTranscludeFn (http://test.com:9012/bower_components/angular/angular.js:8332:16)
    at controllersBoundTransclude (http://test.com:9012/bower_components/angular/angular.js:9108:20)
    at http://test.com:9012/library/ui-select/dist/select.js:1023:9
    at invokeLinkFn (http://test.com:9012/bower_components/angular/angular.js:9580:9)
    at nodeLinkFn (http://test.com:9012/bower_components/angular/angular.js:9066:11)

--- a/src/uiSelectChoicesDirective.js
+++ b/src/uiSelectChoicesDirective.js
@@ -8,6 +8,9 @@ uis.directive('uiSelectChoices',
     replace: true,
     transclude: true,
     templateUrl: function(tElement) {
+      // Needed so the uiSelect can detect the transcluded content
+      tElement.addClass('ui-select-choices');
+
       // Gets theme attribute from parent (ui-select)
       var theme = tElement.parent().attr('theme') || uiSelectConfig.theme;
       return theme + '/choices.tpl.html';

--- a/src/uiSelectMatchDirective.js
+++ b/src/uiSelectMatchDirective.js
@@ -5,6 +5,9 @@ uis.directive('uiSelectMatch', ['uiSelectConfig', function(uiSelectConfig) {
     replace: true,
     transclude: true,
     templateUrl: function(tElement) {
+      // Needed so the uiSelect can detect the transcluded content
+      tElement.addClass('ui-select-match');
+
       // Gets theme attribute from parent (ui-select)
       var theme = tElement.parent().attr('theme') || uiSelectConfig.theme;
       var multi = tElement.parent().attr('multiple');

--- a/test/select.spec.js
+++ b/test/select.spec.js
@@ -967,6 +967,7 @@ describe('ui-select tests', function() {
     expect(function() {
       compileTemplate(
         '<ui-select ng-model="selection.selected"> \
+          <ui-select-match></ui-select-match> \
           <ui-select-choices></ui-select-choices> \
         </ui-select>'
       );

